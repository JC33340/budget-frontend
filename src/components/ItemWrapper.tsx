type ItemWrapperType = {
    children: React.ReactNode;
};

const ItemWrapper = ({ children }: ItemWrapperType) => {
    return (
        <div
            className="border-2 border-light-gray p-4 rounded-lg w-full h-full"
            data-testid="itemWrapper-testid"
        >
            {children}
        </div>
    );
};

export default ItemWrapper;
