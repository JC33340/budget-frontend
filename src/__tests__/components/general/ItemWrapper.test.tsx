import ItemWrapper from '../../../components/ItemWrapper';
import { render, screen } from '@testing-library/react';
import '@testing-library/jest-dom';

test('snapshot test', () => {
    render(
        <ItemWrapper>
            <p>test text</p>
        </ItemWrapper>
    );
    const itemWrapper = screen.getByTestId('itemWrapper-testid');
    expect(itemWrapper).toMatchSnapshot();
});
