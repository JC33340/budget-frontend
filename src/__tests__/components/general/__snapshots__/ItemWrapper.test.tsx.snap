// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot test 1`] = `
<div
  class="border-2 border-light-gray p-4 rounded-lg w-full h-full"
  data-testid="itemWrapper-testid"
>
  <p>
    test text
  </p>
</div>
`;
